### DISCUSSION TITLE
Ayşegülün yaşı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#176] [Soru Cevap]
Ayşegül 32 yaşındadır ve kardeşinin 4 katı yaşındadır. Kaç yaşında kardeşinin iki katı yaşında olacaktır?

**Comment by alicinar11** [comment]
Ayşegül 32 yaşındayken kardeşi 8 yaşındadır, bundan 16 yıl sonra Ayşegül 48, kardeşi ise 24 yaşında olur yani kardeşinin iki katı yaşında olması için Ayşegül'ün 48 yaşında olması gerekir.
